info/v2/conversion.go

```diff
@@ -127,6 +127,7 @@ func ContainerStatsFromV1(spec *v1.ContainerSpec, stats []*v1.ContainerStats) []
 				stat.Filesystem = &FilesystemStats{
 					TotalUsageBytes: &val.Filesystem[0].Usage,
 					BaseUsageBytes:  &val.Filesystem[0].BaseUsage,
+					InodeUsage:      &val.Filesystem[0].Inodes,
```

Comment:
I think that is what actually happens.  &val.Filesystem[0].Inodes may be nil if we skipped collection.  In that case, it should also set InodeUsage to nil as well?